azure_media_services/templates/player.html

```diff
@@ -2,90 +2,87 @@
-                        <li class="amp-menu-header">Transcripts</li>
-                        <li class="vjs-menu-item vjs-selected outline-enabled-control"
-                            tabindex="-1"
-                            role="menuitemcheckbox"
-                            aria-live="off"
-                            aria-label="Transcripts Off"
-                            aria-checked="true"
-                        >
-                            Off
-                        </li>
+  <div class="downloads-container">
+    <ul class="wrapper-downloads-custom wrapper-downloads">
 
-                        {% for transcript in captions %}
-                        <li class="vjs-menu-item outline-enabled-control"
-                            role="menuitemcheckbox"
-                            aria-live="off"
-                            tabindex="-1"
-                            aria-label="{{ transcript.label }}"
-                            aria-checked="false"
-                            data-srclang="{{ transcript.srclang }}"
-                            data-src="{{ transcript.src }}"
-                        >
-                            {{ transcript.label }}
-                            <span class="vjs-control-text"> </span
-                        </li>
-                      {% endfor %}
-                    </ul>
-                </div>
-                <span class="vjs-control-text">Transcripts</span>
+      {% if share %}
+      <li class="share-button video-download-button-custom">
+        <a role="button" href="#" class="js-share-button btn">{% trans 'Share' %}</a>
+        <div class="share-popup-callout is-hidden js-share-popup">
+          <div class="share-callout-content">
+            <div class="share-this-lesson-text">{% trans 'Share this lesson' %}</div>
+            <hr class="separatorLineEmbed">
+            <div class="embed-content">
+              <label for="txtContentEmbed" class="title-lesson-embed bold">{% trans 'Embed this lesson' %}</label>
+              <div class="text-content-embed">
+                <textarea id="txtContentEmbed" readonly="readonly" data-url="{{ embed_url }}"></textarea>
+              </div>
+              <div class="size-embed">
+                <label for="ddlSizeEmbed" class="text-size-embed">{% trans 'Size' %}</label>
+                <select class="ddlSizeEmbed" id="ddlSizeEmbed" >
+                  <option data-width="426" data-height="472" value="small" selected>426 x 472 pixels</option>
+                  <option data-width="636" data-height="480" value="default">636 x 480 pixels</option>
+                  <option data-width="814" data-height="664" value="large">814 x 664 pixels</option>
+                </select>
+              </div>
             </div>
+            <div class="embed-tou">
+              The player is governed by this <a href="https://mva.microsoft.com/license" target="_blank">license</a>.
```

Comment:
This shouldn't reference https://mva.microsoft.com/license since it's language is specific to MVA as is the header/footer.  Perhaps WWL should have a generic license for embeddable components?

Reply:
Yes, we need to confirm from jeff and anupam. we will fix it later.